cloudbridge/cloud/providers/gce/services.py

```diff
@@ -870,14 +834,9 @@ def get(self, volume_id):
         """
         Returns a volume given its id.
         """
-        try:
-            return GCEVolume(self.provider,
-                             self.provider.parse_url(volume_id).get_resource())
-        except googleapiclient.errors.HttpError as http_error:
-            # If the volume is not found, the API will raise
-            # googleapiclient.errors.HttpError.
-            cb.log.warning(
-                "googleapiclient.errors.HttpError: {0}".format(http_error))
+        vol = self.provider.get_resource('disks', volume_id)
+        if vol:
```

Comment:
nit: "return return GCEVolume(self.provider, vol) if vol else None" fits in 1 line, right?

Similarly for bucket, network, region, image, and type (not snapshot and instance) if the short form fits in 1 line.

Reply:
Done.